### DISCUSSION TITLE
Exclude Users from Triggering Notifications

### DISCUSSION THREAD

**Discussion by evolkmann** [community/community#3695] [Mobile]
> I just activated the new notifications settings and they are great, thanks for that first of all!

**Proposal**

Allow me to maintain a list of users. When anyone of them triggers a notification (for example by requesting a review from me), no notification is sent. This could also be a global setting in my profile and affect all types of notifications, not only for the mobile app.

**Use Case**

We have a bot user that creates automatic PRs for dependency updates. It requests a review from a configured user. 

I do not need notifications for these PRs, because I know they are created frequently, I just want to see them in the Dashboard whenever I visit Github.

**Comment by brianlovin** [comment]
Great idea, I have the same problem with Dependabot opening a bunch of PRs and assigning me, and then I wake up with dozens of notifications.
